model/src/main/java/io/syndesis/model/integration/Integration.java

```diff
@@ -73,7 +73,7 @@ default Kind getKind() {
 
     Optional<String> getIntegrationTemplateId();
 
-    Optional<String> getUserId();
+    String getUserId();
```

Comment:
Doesn't this mean the UI will have to send the userId every time with the request?

Reply:
Yes think so, which is why I asked for @gashcrumb's review to make sure this is ok.

Reply:
Should we trust the username provided by the UI? Can we add a default method so its populated from Spring Security if not set already?

Reply:
Yeah I like the idea of setting it on the backend. If we do that we should not even have the UI worry about it.

Reply:
We can add that as a default value, but would require muddying dependencies by adding spring secuity as a dependency to model.